### DISCUSSION TITLE
Issue with new relationship picker field

### DISCUSSION THREAD

**Discussion by benaunt** [Budibase/budibase#11766] [Help] [answered]
Hi,

We have the following scenario.

1. A client table
2. An address table

The tables have a one client to many relationship with the address table

Within BB on a form linked to the client table, we have the relationship field connected to the address table. The address relationship picker has a filter to only show addresses that map to a variable client ID.

This works until the number of addresses is above 100, after that the drop down show's no entries.

I understand that the relationship field should have a row limit, but if you are using a filter it should be pre-filtering the results before loading them in.

Thanks

**Comment by ConorWebb96** [comment, marked as answer]
Hey @benaunt,

With @melohagan recent PRs you have enhanced search functionality. https://github.com/Budibase/budibase/pull/11639. This should get around this limit you are experiencing. 

I hope this helps.
